plugins/outputs/cloudwatch_logs/cloudwatch_logs.go

```diff
@@ -0,0 +1,434 @@
+package cloudwatch_logs
+
+import (
+	"fmt"
+	"log"
```

Comment:
Instead of using the log package can you update this to use the Telegraf logger:
```
Logger Telegraf.Logger `toml:"-"`
```
(in tests, you can do myPlugin.Log = testutil.Logger{})

Reply:
Hello, sure, will fix that.